llvm/utils/mlgo-utils/mlgo/corpus/extract_ir_lib.py

```diff
@@ -316,6 +316,27 @@ def make_spec(obj_file: str):
     return [make_spec(path) for path in paths]
 
 
+def load_bazel_aquery(aquery_json, obj_base_dir: str, output_dir: str):
+    """Creates an object file array by looking at the JSON output of bazel aquery.
+
+    Args:
+      aquery_json: The JSON-formatted output of the bazel aquery command for
```

Comment:
Is there a link we can put here for what that format is?

Reply:
It's a JSON version of [this proto](https://github.com/bazelbuild/bazel/blob/3f93d377d036d773fd505a18e084425a00fb94ea/src/main/protobuf/analysis_v2.proto#L25). I've added a comment.